File: solutions/jose-rivas/python/p-05-functions/exercise-01.py
# Fonction pour additionner deux nombres
def addition(a, b):
    return a + b

try:
    valeurs = input("Entrez deux nombres : ").split()

    a = int(valeurs[0])
    b = int(valeurs[1])

    resultat = addition(a, b)

    print("La somme est :", resultat)

except:
    print("Erreur : veuillez entrer deux nombres entiers.")

File: solutions/jose-rivas/csharp/p-05-functions/exercise-01.cs
using System;

class Program
{
    // Fonction pour additionner deux nombres
    static int Addition(int a, int b)
    {
        return a + b;
    }

    static void Main()
    {
        try
        {
            Console.Write("Entrez deux nombres : ");
            string[] valeurs = Console.ReadLine().Split();

            int a = int.Parse(valeurs[0]);
            int b = int.Parse(valeurs[1]);

            int resultat = Addition(a, b);

            Console.WriteLine("La somme est : " + resultat);
        }
        catch
        {
            Console.WriteLine("Erreur : veuillez entrer deux nombres entiers.");
        }
    }
}